Automattic/wp-calypso
# Woo Express: Remove unused component

Related to #75560

## Proposed Changes

* We no longer use the `EcommercePlanFeatures` component so this removes the code files. The component instances were removed in #75572 

## Testing Instructions



* Switch to this PR
* Test the `/plans` page on a site with all the Woo Express plans, including an expired trial site. Make sure everything still works and there are no fatal errors.

## Pre-merge Checklist



- N/A [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- N/A Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- N/A Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- N/A For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
